code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -297,18 +292,16 @@
 	to_chat(src, "<b>Don't invade their worksites, don't steal their resources, don't tell them about the changeling in the toilets.</b>")
 	to_chat(src, "<b>Make sure crew members do not notice you.</b>.")
 
-/*
-	sprite["Default"] = "repairbot"
-	sprite["Mk2 Mousedrone"] = "mk2"
-	sprite["Mk3 Monkeydrone"] = "mk3"
+	//now that the drone has a player, let's pick a sprite.
+	var/drone_icons[0]	//these are our sprite choices
+	drone_icons["Standard"] = "repairbot"
```

Comment:
Its weird that all the sprites are named drone_something, except the default one.

Reply:
i was thinking that too, can change it now

Reply:
changed all references to drone_repairbot